.github/CODEOWNERS

```diff
@@ -0,0 +1,3 @@
+* @JakeLane @pancaspe87 @liamqma
+**/package.json
```

Comment:
I left a quick note in the description, sorry if it wasn't clear.

I noted that it was excluded in the other repo and tracked the previous PR to understand why https://github.com/atlassian-labs/compiled/pull/981

The link mentioned in the other PR doesn't seem to work anymore, but I guess it was to avoid getting bugged by renovate.